notebooks/book2/Fig16_1_(book2)_mlpPriorsDemo.ipynb

```diff
@@ -0,0 +1,998 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    class MLP(nn.Module):
</pre><p>Can you try and use the following style with <code>@nn.compact</code>? It is more flaxy, <code>setup</code> version is more torchy.    https://flax.readthedocs.io/en/latest/notebooks/flax_basics.html#:~:text=class%20SimpleMLP(nn.Module)%3A</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>